logagg/collect/README.md

```diff
@@ -0,0 +1,67 @@
+### Handling the logs
+
+- To get the nginx logs in json format we have to modify the `nginx.conf` file in `/etc/nginx` directory.
+
+- Modify the `Logging Settings` section in the `nginx.conf` file by adding the below lines:
+```bash
+log_format  json  '{'
+                    '"remote_addr": "$remote_addr",'
+                    '"remote_user": "$remote_user",'
+                    '"timestamp": "$time_iso8601",'
+                    '"request": "$request",'
+                    '"status": "$status",'
+                    '"request_time": "$request_time",'
+                    '"body_bytes_sent": "$body_bytes_sent",'
+                    '"http_referer": "$http_referer",'
+                    '"http_user_agent": "$http_user_agent",'
+                    '"http_x_forwarded_for": "$http_x_forwarded_for"'
+                    '}';
+```
+
+- Modify the line: `access_log /var/log/nginx/access.log;` to `access_log /var/log/nginx/access.log json;`
+
+
+- To test nginx config file
+```bash 
+nginx -t
+```
+
+- It shows like:
+```bash
+nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
+
+nginx: configuration file /etc/nginx/nginx.conf test is successful
+```
+
+- Restart the nginx by using command:
+```bash
+/etc/init.d/nginx restart
+```
+
+
```

Comment:
@AnushaKonduru @praneeth36 We don't need more than one new line to separate out sections.